docs/en/usage/java-sdk-user-guide.md

```diff
@@ -1269,3 +1269,16 @@ public class SpringIntegrationTest {
 }
 ```
 
+# Ⅶ. apollo-client customization
+
+## 7.1 ConfigService load balancing algorithm
+
+> from version 2.1.0
+
+For satisfy user's different demands on ConfigService load balancing algorithm when user use apollo-client, we provide **spi** in version 2.1.0
```

Comment:
```suggestion
To satisfy users' different demands on ConfigService load balancing algorithm when using apollo-client, we provide **spi** since version 2.1.0
```